test/nature_test_case.py

```diff
@@ -42,7 +42,7 @@ class QiskitNatureTestCase(unittest.TestCase, ABC):
     def setUp(self) -> None:
         warnings.filterwarnings("default", category=DeprecationWarning)
         warnings.filterwarnings("ignore", category=DeprecationWarning, module="pyscf")
-        warnings.filterwarnings(action='ignore', category=DeprecationWarning, module='.*drivers.*')
+        warnings.filterwarnings(action="ignore", category=DeprecationWarning, module=".*drivers.*")
```

Comment:
Doesn't this end up suppressing all such warnings from any code under drivers, including the new relocated ones. If we ended up using any deprecated code there we would want to fix it and hence see any messages in the build output. Should it have another line to turn things on again if its drivers.second_quantization.*? 

I note that there are still deprecation messages appearing in the CI log from HFMethod and InitialGuess enums